src/lib.rs

```diff
@@ -130,6 +130,9 @@ pub enum SpliceCommand {
         reserved: u8,
         splice_detail: SpliceInsert,
     },
+    TimeSignal {
+        splice_time: SpliceTime
```

Comment:
This works, but I wonder if it should be `Timed` - I initially mistook that `SpliceTime` was analogous to `splice_time`, which it doesn't appeat to be.

Reply:
No, it doesn't work like that. I have much Rust to learn.